### DISCUSSION TITLE
I want to scan my React codebase for non-Tailwind classnames

### DISCUSSION THREAD

**Discussion by alexgleason** [tailwindlabs/tailwindcss#10305] [Help] [answered]
I'm converting a legacy codebase to Tailwind. The process so far has been to delete scss files one at a time, and Ctrl+F trying to find them in the codebase. But since (S)CSS can do stuff like `.a > .b` and `&--bem`, it's easy to miss. Instead I wonder if I can analyze the TS/TSX codebase for classNames that aren't Tailwind.

**Comment by felquis** [comment, marked as answer]
I think `eslint-tailwind-plugin` can help you with the HTML part at least.

With the style definitions it's hard to tell, some tailwind projects doesn't have ANY custom css defined.

**Reply by alexgleason** [reply]
[no-custom-classname](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/HEAD/docs/rules/no-custom-classname.md) is exactly what I was looking for. Thanks!
